src/components/Contact/Contact.jsx

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import { Row, Col } from 'reactstrap';
+import "./Contact.css";
+import { IconContext } from "react-icons";
+import { graphql, useStaticQuery } from 'gatsby';
+import Img from 'gatsby-image';
+import {
+  FaGithubSquare,
+  FaFacebookSquare,
+  FaTwitterSquare,
+  FaDiscord,
+  FaYoutubeSquare,
+  FaRedditSquare,
+  FaGooglePlusSquare,
+  FaComments
+} from "react-icons/fa";
+
+
+export default props => {
+  const data = useStaticQuery(graphql`
+    query myImages2{
+      historyImg: file(relativePath: {eq: "images/3.jpg"}){
+        childImageSharp{
+          fluid(maxWidth: 300){
+            ...GatsbyImageSharpFluid_tracedSVG
+          }
+      }
+    },
+    modding: file(relativePath: {eq: "images/9.jpg"}){
+      childImageSharp{
+        fluid(maxWidth: 300){
+          ...GatsbyImageSharpFluid
+        }
+      }
+    },
+    community: file(relativePath: {eq: "images/8.jpg"}){
+      childImageSharp{
+        fluid(maxWidth: 300){
+          ...GatsbyImageSharpFluid_tracedSVG
+        }
+      }
+    }
+  }
+`);
+
+  return (
+    <div className="contact">
+      <div className="contact_bg">
+        <div className="contact__bg-overlay">
+          <Row className="justify-content-center">
+            <Col md="10" className="justify-content-center">
+              <h1>Contact Us</h1>
+            </Col>
+          </Row>
+        </div>
+      </div>
+      <section className="contact__details py-5 mt-6e">
+        
+      <Row>
+           <IconContext.Provider value={{ size: "6em"}}>
```

Comment:
All of the icons are too large, you can have a look at other games' contact pages to get a gist.

Reply:
Resized 👍